doc/changes/devel/12583.apichange.rst

```diff
@@ -0,0 +1,2 @@
+:attr:`mne.Info.ch_names` will now return an empty list instead of raising a `KeyError` if no channels
```

Comment:
@larsoner @drammock This reference apparently doesn't work -- do you know why? ch_names is a property of Info

Reply:
AFAIK it should work: https://www.sphinx-doc.org/en/master/usage/domains/python.html#role-py-attr

Reply:
Maybe it's something about the namespace ...

The doc build fails with:

> /home/circleci/project/doc/<rst_prolog>:30: WARNING: py:attr reference target not found: mne.Info.ch_names [ref.attr]